data/json/items/gun/shot.json

```diff
@@ -382,6 +388,7 @@
         "name": { "str": "Remington 1100 competition" },
         "description": "This semi-automatic shotgun features a self compensating gas system that will feed a wide array of shells while also reducing recoil.  Introduced in 1963, it is favored by law enforcement, hunters and competition shooters, and has been the best-selling autoloading shotgun in U.S. history.  This is the nickel finished, teflon coated competition model, with a full length magazine tube and 30 inch barrel.",
         "ascii_picture": "remington1100"
+		"barrel_length": "762 mm",
```

Comment:
can variants define barrel_length? glancing at c++ side doesn't look like it

Reply:
They cannot

Reply:
So, what is better, delete this lines, or leave for the time?

Reply:
It likely won't pass tests if you leave random bits of unused json, unless the specific json loader allows for it (don't count on it, as most don't)